packages/framework/agent-scheduler/src/agent.ts

```diff
@@ -35,7 +36,7 @@ export interface IAgentSchedulerEvents extends IEvent {
 /**
  * Agent scheduler distributes a set of tasks/variables across connected clients.
  */
-export interface IAgentScheduler extends IProvideAgentScheduler, IEventProvider<IAgentSchedulerEvents> {
+export interface IAgentScheduler extends IProvideAgentScheduler, IEventProvider<IAgentSchedulerEvents>, IFluidLoadable {
```

Comment:
Probably would have been better to leave this off `IAgentScheduler` and only put it on `AgentScheduler` -- prior to this change the `IAgentScheduler` was only about usage of the scheduler and not about the data store "stuff".

With this change, it's possible that components that were only intended to be "users" of the scheduler (i.e. were passed an `IAgentScheduler` but not an `AgentScheduler`) can now pin a reference to the handle.  Maybe consider moving it.

Reply:
Can you open an ADO and assign it to me?